composer.json

```diff
@@ -11,7 +11,8 @@
     ],
     "require": {
         "php": "~5.6|~7.0",
-        "ezsystems/ezpublish-kernel": "^6.0|^7.0"
+        "ezsystems/ezpublish-kernel": "^6.0|^7.0",
+        "netgen/tagsbundle": "^3.2"
```

Comment:
We should probably check that TagsBundle is active too. Something like: https://github.com/netgen/TagsBundle/blob/3.2-release/bundle/DependencyInjection/NetgenTagsExtension.php#L26-L30

Or thrown an exception in content methods working with tags if the bundle is not active.

I'd rather have second one, since we do not want to force others to use TagsBundle just to use site API.

Reply:
Ok, then `netgen/tagsbundle` goes to the suggest section. Any input on the minimal version we should support?

Reply:
You can use `\Netgen\TagsBundle\Version::VERSION_ID >= 30000` condition.